code/model/entity/abstract.php

```diff
@@ -269,10 +269,14 @@ public function setProperty($name, $value, $modified = true)
      */
     public function hasProperty($name)
     {
+        if (empty($name)) {
+            return false;
```

Comment:
I can refactor, however, https://m.youtube.com/watch?v=GtB5DAfOWMQ is a really interesting video on why that's a bad idea. I've found it makes code much easier to read. 

Returning early reduces the cognitive load and reduces the need for complex if statements. 


> On 9 May 2016, at 01:45, Johan Janssens <notifications@github.com> wrote:
> 
> In code/model/entity/abstract.php:
> 
> > @@ -269,10 +269,14 @@ public function setProperty($name, $value, $modified = true)
> >       */
> >      public function hasProperty($name)
> >      {
> > +        if (empty($name)) {
> > +            return false;
> @oligriffiths Return statements should only be at the end of the function.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>